sencha-workspace/SlateAdmin/app/view/people/details/Courses.js

```diff
@@ -38,23 +38,41 @@ Ext.define('SlateAdmin.view.people.details.Courses', {
         xtype: 'grid',
         border: false,
         viewConfig: {
-            emptyText: 'No courses for selected term'
+            emptyText: 'No courses for selected term',
+            getRowClass: function(record){
+                const now = new Date();
+                let isActive = true;
+
+                if (record.get('StartDate')) {
+                    let startTime = new Date(Date.parse(record.get('StartDate')));
```

Comment:
@nbey this should happen at the model layer, and already does in the `slate-core-data` package's version of this model

Since we're determining a state here that's based wholly on information in the model and doesn't need to be redone outside that data changing, and because the information isn't UI-specific (i.e. its whether the enrollment is active, not what the CSS class should be) we should define `isInactive` as a "virtual" field. `slate-cbl` has a lot of really good examples of this: https://github.com/SlateFoundation/slate-cbl/blob/198ee2c052616159d35a4d15476885d31e689ed9/sencha-workspace/packages/slate-cbl/src/model/tasks/StudentTask.js#L214

Also, this isn't a binary state of inactive-vs-active. We want to add the `inactive` CSS class to enrollments that have start/end dates that the current date is outside of, but an enrollment without that isn't necessary active (depends on the current term and term structures). We might want to have an `isCurrent` determination available inside the participant model at some point, but that will take some work. For now, remove the else case and don't add any CSS class if we're not adding the `.participant-inactive` one (switch the terms too, we want CSS state classes to be prefixed with their context and suffixed with their state like the framework does)